lambda/translclass.ml

```diff
@@ -39,28 +39,16 @@ let layout_tables = Lambda.Pvalue Pgenval
 
 let lfunction ?(kind=Curried {nlocal=0}) ?(region=true) ?(ret_mode=alloc_heap) return_layout params body =
   if params = [] then body else
-<<<<<<< HEAD
   match kind, body with
   | Curried {nlocal=0},
     Lfunction {kind = Curried _ as kind; params = params';
                body = body'; attr; loc; mode = Alloc_heap; ret_mode; region}
+    (* XXX mshinwell: 5.2.0 also has:
+         when attr.may_fuse_arity && ...
+    *)
```

Comment:
We should do the same for consistency. (Stephen's RFC doesn't specify syntactic arity for class methods, but the change we made upstream is more consistent with the spirit of the RFC, and we might as well do what we did upstream.)